packages/nextjs/app/page.tsx

```diff
@@ -1,16 +1,26 @@
+"use client";
+
 import Link from "next/link";
 import type { NextPage } from "next";
+import { useAccount } from "wagmi";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
 
 const Home: NextPage = () => {
+  const { address: connectedAddress } = useAccount();
+
   return (
     <>
       <div className="flex items-center flex-col flex-grow pt-10">
         <div className="px-5">
-          <h1 className="text-center mb-8">
+          <h1 className="text-center">
             <span className="block text-2xl mb-2">Welcome to</span>
             <span className="block text-4xl font-bold">Scaffold-ETH 2</span>
           </h1>
+          <div className="flex justify-center items-center space-x-2">
```

Comment:
NITPICK:  Should we add a line above this ? :

```
{/* Can be moved to a separate client component instead of using "use client" for whole page */}

```

Reply:
I won't personally add it. Seems to specific to NextJS!